baseProject/src/main/webapp/resources/js/dev/+alerts.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @name alerts
+ * @memberof jQuery.flatfindr
+ * @namespace jQuery.flatfindr.alerts
+ *
+ * @param  {Object} window   the window as you know it
+ * @param  {Object} document the document element
+ * @param  {jQuery} $
+ * @param  {jQuery} $view    the view, defaults to the body element
+ * @param  {Object} option
+ */
+jQuery.flatfindr.alerts = function (window, document, $, $view, option) {
+
+
+  $.flatfindr.bits.addAutoloc('#city');
+
+
+  $('form').on('submit', function(e) {
+    e.preventDefault();
+    validateType();
+    this.submit();
+  });
+
+
+  $('.deleteButton').on('click', deleteAlert);
+
+
+  /**
+   * Delete previously set alerts.
+   * @private
+   */
+  function deleteAlert() {
+    var id = $(this).attr("data-id");
+    $.get("/profile/alerts/deleteAlert?id=" + id, function(){
+      $("#alertsDiv").load(document.URL + " #alertsDiv");
+    });
+  }
+
+
+
+  /**
+   * @private
+   */
+  function validateType() {
+
+		var room = document.getElementById('room');
+		var studio = document.getElementById('studio');
+		var house = document.getElementById('house');
+		var neither = document.getElementById('neither');
+		var r = document.getElementById('r');
+		var s = document.getElementById('s');
+		var h = document.getElementById('h');
+		var ras = document.getElementById('ras');
+		var rah = document.getElementById('rah');
+		var sah = document.getElementById('sah');
+		var all = document.getElementById('all');
+
+    neither.checked = false;
```

Comment:
Forgot to mention that someone forgot to reset 'neither' as well here. The problem was: If the form is submitted the first time with no roomtype set, resp. neither = true, the page indeed loads correctly with an error of "At least one roomtype... blah" as neither is set to true. But, when the unlucky user now hits a roomtype, e.g. room = true and submits the form again, neither is still set to true and roomtype is still an invalid input.